src/utils/chunkArray.ts

```diff
@@ -0,0 +1,10 @@
+export function chunkArray<T>(array: T[], chunkSize: number): T[][] {
```

Comment:
Good idea to define an infer there `T` but it is not used anywhere in the code :/

Reply:
Yes, I did not add because it seems to be inferred:
![image](https://github.com/user-attachments/assets/f2be628e-6ca9-4ade-9617-39897c89d1a3)